tests/test_stdext.ml

```diff
@@ -0,0 +1,35 @@
+let is_backup_file () =
+  let check ~input ~expected =
+    let name = "Path.is_backup_file " ^ input in
+    let actual = Dune_release.Stdext.Path.is_backup_file input in
+    Alcotest.(check bool) name expected actual
+  in
+  check ~input:"" ~expected:false;
+  check ~input:"fooooooooooooo" ~expected:false;
+  check ~input:"fooooooooo#" ~expected:false;
+  check ~input:"#fooooooooo#" ~expected:true;
+  check ~input:"foooooooooooo~" ~expected:true
+
+let find_files () =
+  let check ~name ~paths ~name_wo_ext ~expected =
+    let paths = List.map Fpath.v paths in
+    let expected = List.map Fpath.v expected in
+    let actual = Dune_release.Stdext.Path.find_files ~name_wo_ext paths in
+    let open Alcotest in
+    let open Alcotest_ext in
+    (check (list path)) name expected actual
+  in
+  check ~name:"Path.find_files empty" ~paths:[] ~name_wo_ext:"" ~expected:[];
+  check ~name:"Path.find_files does not contain" ~name_wo_ext:"foo"
+    ~paths:[ "aaa"; "bbb"; "#foo#"; "foo~"; "foo.md.swp" ]
```

Comment:
For accuracy, `vi(m)` swp files are dot files, i.e. it should be `.foo.md.swp`

Reply:
done